Ports/ja2/package.sh

```diff
@@ -0,0 +1,20 @@
+#!/usr/bin/env -S bash ../.port_include.sh
+port=Jagged-Alliance-2
+version=0.15.x
+depends=("SDL2")
+workdir=ja2-stracciatella-0.15.x
+files="https://github.com/ja2-stracciatella/ja2-stracciatella/archive/refs/heads/${version}.zip ja2-stracciatella-0.x.15.zip 178375de4859d16a76276c781455bf48d3fa862841387c8aa6cfa4162f4f0ca4"
```

Comment:
`0.x.15`?

Reply:
I must have missed reformatting in the patch, sorry for that, will fix it.
0.x.15 is a typo, will fix it as well.

Thx